include/swift/AST/Decl.h

```diff
@@ -3575,6 +3575,8 @@ class ProtocolDecl : public NominalTypeDecl {
   /// created yet.
   void createGenericParamsIfMissing();
 
+  GenericSignature *getRequirementSignature();
```

Comment:
Please put a doc comment here so it's clear to the reader what the requirement signature actually is. You can basically use the text of your commit message here.